src/utils/network.js

```diff
@@ -24,6 +24,43 @@ export const getFallbackProvider = (networkId) => {
     urls = [
       "https://arb1.arbitrum.io/rpc"
     ];
+  // +
+  } else if (networkId === 19) {
+    // Songbird Canary-Network
+    urls = [
+      "https://songbird-api.flare.network/ext/C/rpc"
+    ];
+  } else if (networkId === 14) {
+    // Flare Mainnet
+    urls = [
+      "https://flare-api.flare.network/ext/C/rpc"
+    ];
+  } else if (networkId === 8453) {
+    // Base
+    urls = [
+      "https://rpc.notadegen.com/base"
```

Comment:
@nidz-the-fact Let's use the official RPC node, because we can be sure that official RPC node will always exist (these 3rd party nodes may shut down at some point).

The official RPC for Base is `https://mainnet.base.org`.

Do the same for other RPCs in this file. Check what are the official RPC nodes in the docs of each chain.

Reply:
Ok